intro/fonts/index.html

```diff
@@ -78,9 +78,11 @@ <h2><a id="rendering-fonts" class="js"></a>Rendering fonts in Pygame</h1>
         <ul>
             <li><code>text</code>, which is a string with the text you want to render;</li>
             <li><code>antialias</code>, which is a boolean indicating whether you want your text to have smooth edges;</li>
-            <li><code>color</code>, which can be a <code>tuple[int, int, int]</code> or a string.</li>
-            <li><code>background</code>, which is a color value and can be a <code>tuple[int, int, int]</code> or a string.</li>
+            <li><code>color</code>, which is usually a <code>tuple[int, int, int]</code> or a string;</li>
+            <li><code>background</code>, which is usually a <code>tuple[int, int, int]</code> or a string.</li>
         </ul>
+        Note: for all supported color formats, see <a href="intro/colors/">Colors in pygame</a>.
```

Comment:
Shouldn't it have a leading forward slash and does it need the trailing one?

Reply:
Oh yeah, probably right about the leading forward slash. Trailing one is optional (that's why each page is in it's own folder with an `index.html`)